diff --git a/src/TwitterPostManager.php b/src/TwitterPostManager.php
index bc62fe5..4ea7a2f 100644
--- a/src/TwitterPostManager.php
+++ b/src/TwitterPostManager.php
@@ -74,7 +74,7 @@ public function uploadMedia(array $paths) {

     foreach ($paths as $path) {
       // Upload the media from the path.
-      $media = $this->client->upload('media/upload', ['media' => $path]);
+      $media = $this->client->upload('media/upload', ['media' => $path, 'media_type' => 'video/mp4'], TRUE);

       // The response contains the media_ids to attach the media to the post.
       $media_ids[] = $media->media_id_string;
